designs/penelope/src/shape.mjs

```diff
@@ -184,6 +221,7 @@ export function BuildMainShape(part, frontPart) {
   paths.waist1 = waistCurve.translate(0, 10).attr('class', 'lining dashed')
 
   if (iteration >= 100) {
+    console.log( 'Too many iterations trying to make it fit!' )
```

Comment:
Although I don't know how the behavior of `throw` is different, you might want to consider changing this to use `log.error()`. After adding `log` to the part.shorthand() section at the top, the change would be:
```js
log.error( 'Too many iterations trying to make it fit!' )
```
This will cause the pattern to halt and crash, with the error message in the Logs section of the lab.